05-networks/index.ts

```diff
@@ -0,0 +1,43 @@
+import * as http from "http";
+import * as url from "url";
+import { StringDecoder } from "string_decoder";
+import router from "./user/routes";
+import { IncomingMessageWithBody } from "./models/inccoming-message-with-body.interface";
+
+const server = http.createServer(
+  (req: IncomingMessageWithBody, res: http.ServerResponse) => {
+    const parsedUrl = url.parse(req.url!, true);
```

Comment:
Just out of curiosity: I wonder why it was needed to assert that req.url exists? A request should have a url always, by default.